### DISCUSSION TITLE
Snapper requires smaller limits on disk usage by default. (and also a better explanation of how to configure it)

### DISCUSSION THREAD

**Discussion by Ricardox1** [SpiralLinux/SpiralLinux-project#440] [General]
I am not a new Linux user, but I am new to using Spirallinux and BTRFS, on other computers I use LinuxMint, but the compression on disk and ram by default in Spirallinux was attractive to me to try it.

However, I ran into an issue of running low on disk space, for no apparent reason. After investigating, I “discovered” that the cause was Snapper, which had been saving a bunch of snapshots for over 6 months. By deleting these old snapshots the disk space reappeared.

Therefore, my suggestions in this regard are two:
1 – Put tighter limits on Snapper Snapshots.
2 – Better communicate how to configure it. (Investigating on my own I found the configuration files in etc/snapper/config)

The point is that unless the user is a Nerd, they will encounter a problem that is difficult to solve, so I think that configuring by default that Snapper stores fewer Snapshots may not be a bad idea.

**Comment by geckolinux** [comment]
Hi there, thanks for the suggestions.

Regarding `#1`, I'm not sure which version of SpiralLinux you originally installed from, but the 12.231120 release changed the default disk space max usage for Snapper to `SPACE_LIMIT=0.2` .

Regarding `#2`, the **Snapper GUI (root)** app allows for configuring some aspects of the configuration via a GUI:
![image](https://github.com/SpiralLinux/SpiralLinux-project/assets/2692138/e4d1ce77-5f93-46c7-9ee5-e8ec31462c43)

![image](https://github.com/SpiralLinux/SpiralLinux-project/assets/2692138/d821214a-355e-4ce2-a228-78ca04d69a99)

**Reply by Ricardox1** [reply]
Thanks for your reply.
